src/flick/search/views.py

```diff
@@ -85,14 +67,14 @@ def get_lsts_by_name(self, query):
     def get(self, request, *args, **kwargs):
         query = request.query_params.get("query")
         print(f"query: {query}")
+        tag_lst = request.GET.getlist("tags", [])
```

Comment:
oh hm is there a difference between `request.GET` and `request.query_params`? if one is better than the other, could we make this consistent